packages/truffle/scripts/ganache.js

```diff
@@ -25,4 +25,8 @@ server.listen(PORT, async (err) => {
   });
 
   console.log(accounts);
+
+  if(!process.env.MNEMONIC_PHRASE) { 
+    console.log("mnemonic used: "+provider.getOptions().wallet?.mnemonic); 
```

Comment:
Is this file not linted?
```suggestion
    console.log("mnemonic used: " + provider.getOptions().wallet?.mnemonic); 
```

Reply:
It was linted, I guess lint did not catch this. I have probably made this mistake in other places too.

Reply:
Oh, maybe this file is not linted.